### ISSUE TITLE
Address for pet repos not being set properly

### ISSUE THREAD

**Issue by q5sys** [puppylinux-woof-CE/woof-CE#642]
Just tested with using Latest commit 451bcc5, not sure when issue arose. 

Steps to recreate:

1) ./merge2out
2) Option 3
3) Option 3
4) Option 1
5) Option 2

> Choices:
> Host architecture:     x86_64
> Target architecture:   x86_64
> Compatible-distro:     slackware64
> Compat-distro version: 14.1

./0setup

Downloading Packages-puppy-slacko6414.1-official...
--2015-11-23 18:04:24--  http://distro.ibiblio.org/puppylinux/Packages-puppy-slacko6414.1-official
Resolving distro.ibiblio.org (distro.ibiblio.org)... 152.19.134.43
Connecting to distro.ibiblio.org (distro.ibiblio.org)|152.19.134.43|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 61088 (60K) [application/octet-stream]
Saving to: 'remotedbfile'

100%[==================================================>] 61,088       287KB/s   in 0.2s   

2015-11-23 18:04:24 (287 KB/s) - 'remotedbfile' saved [61088/61088]

...files are the same.

However @

> Press any printable char to check if PET pkgs in local db files
> actually exist on Internet host site.
> WARNING: THIS TAKES A LONG TIME, RECOMMEND BYPASS
> or ENTER only to bypass check: 

Spider mode enabled. Check if remote file exists.
--2015-11-23 18:05:26--  http://distro.ibiblio.org/puppylinux//915resolution-0.5.3-patched20110617-x86_64.pet
Resolving distro.ibiblio.org (distro.ibiblio.org)... 152.19.134.43
Connecting to distro.ibiblio.org (distro.ibiblio.org)|152.19.134.43|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
Remote file does not exist -- broken link!!!

Script searching @ http://distro.ibiblio.org/puppylinux//915resolution-0.5.3-patched20110617-x86_64.pet
File actually @ http://distro.ibiblio.org/puppylinux/pet_packages-slacko6414.1/915resolution-0.5.3-patched20110617-x86_64.pet

I believe the variable DISTRO_DB_SUBNAME is not getting set properly

**Comment by 01micko** [comment]
This is still an issue, saw it today.

**Comment by wdlkmpx** [comment]
This code was removed. I fixed it.

0setup calls fix-puppy-dbs, and the the part to check pet online pkgs is incomplete. findpkgs create the definitive pkg list with a proper logic (complicated code) and you'll notice right away  a pkg is missing... and 1download is the script to check the online pet pkgs

Now 1download displays detailed info about each download. .. and downloads them at the same time. It also has refined logic.
